unified-codes/.env

```diff
@@ -1,14 +1,26 @@
-# Default values
-## Storybook doesn't get .local.env or sub-dir .env files loaded for it. 
-## For storybook usages, enter them here.
-## For local dev, create entries in ./apps/web/.local.env
-## For production entries - these are committed and are in ./apps/web/.env
-
-NX_AUTHENTICATION_URL=http://localhost:9990/auth/realms/unified-codes
-NX_LEGACY_API_PORT=3000
-NX_DATA_SERVICE=http://localhost:4000/graphql
-NX_DATA_SERVICE_PORT=4000
-NX_DGRAPH_ENDPOINT=http://localhost:8080
-NX_CLIENT_SECRET=asecretthing
-NX_CLIENT_ID=unified-codes-data
-NX_RXNAV_ENDPOINT=https://rxnav.nlm.nih.gov/REST
\ No newline at end of file
+## For local development and storybook usages, rename this file as .env.
+
+## NOTE: should not be overridden using local or subdirectory env files,
+## as storybook will not find these files.
+
+AUTHENTICATION_SERVICE_URL=http://localhost
+AUTHENTICATION_SERVICE_PORT=9990
+AUTHENTICATION_SERVICE_REALM=auth/realms/unified-codes
+AUTHENTICATION_SERVICE_AUTH=protocol/openid-connect/token
+AUTHENTICATION_SERVICE_CLIENT_ID=unified-codes-data
+AUTHENTICATION_SERVICE_CLIENT_SECRET=asecretthing
+AUTHENTICATION_SERVICE_GRANT_TYPE=password
+
+DATA_SERVICE_URL=http://localhost
+DATA_SERVICE_PORT=4000
+DATA_SERVICE_GRAPHQL=graphql
+
+DGRAPH_SERVICE_URL=http://localhost
+DGRAPH_SERVICE_PORT=8080
+
+LEGACY_API_URL=http://localhost
+LEGACY_API_PORT=3000
+
+RXNAV_SERVICE_URL=https://rxnav.nlm.nih.gov
+RXNAV_SERVICE_PORT=8080
```

Comment:
Just a minor thing - I don't think rxnav exposes their API with ports explicitly, though I'm also kind of now thinking I should have removed the rxnav code when I removed the endpoint which allowed anyone the ability to test it meaningfully 😅